src/test/examples/import-state.json

```diff
@@ -28,26 +28,26 @@
   ],
   "environments": [
     {
-      "name": "state-visible-environment",
-      "type": "production",
-      "displayName": "Visible"
+      "name": "development",
+      "type": "development",
+      "displayName": "Dev"
     },
     {
-      "name": "state-hidden-environment",
+      "name": "production",
       "type": "production",
-      "displayName": "Hidden"
+      "displayName": "Prod"
```

Comment:
All environments are enabled by default, renaming them to avoid misleading terms such as `visible` or `hidden` cause both environments are enabled and visible